Yousei.SourceGen/ParameterizedGenerator.cs

```diff
@@ -17,12 +17,14 @@ public class ParameterizedGenerator : ISourceGenerator, ISyntaxReceiver
         public void Execute(GeneratorExecutionContext context)
         {
             var attributeSource = GetAttributeSource();
-            context.AddSource("Attribute.cs", attributeSource);
+            context.AddSource("Attribute.g.cs", attributeSource);
```

Comment:
Typo?

Reply:
Do you mean the added ".g"?

Reply:
Yes, from the  sound of it it's not a typo.